README.md

```diff
@@ -5,6 +5,29 @@ The Operator Courier is used to build, validate and push Operator Artifacts.
 It is currently in a pre-alpha stage.
 
 ## Usage
+
+### Command Line Interface
+To install the operator courier project to use from the command line, just install the latest release from [PyPI](https://pypi.org/project/operator-courier/):
+
+```bash
+$ pip3 install operator-courier
+$ operator-courier
```

Comment:
This seems to imply executing `operator-courier` is required to install the package. I suggest leaving it out.

Reply:
updated.